composer.json

```diff
@@ -61,18 +59,13 @@
         ],
         "check-cs": "vendor/bin/ecs check --ansi",
         "fix-cs": "vendor/bin/ecs check --fix --ansi",
-        "phpstan": "php -d memory_limit=1G vendor/bin/phpstan analyse --error-format symplify --ansi",
+        "phpstan": "vendor/bin/phpstan analyse --ansi --error-format symplify",
         "rector": "vendor/bin/rector process --ansi"
     },
     "config": {
         "sort-packages": true,
         "allow-plugins": {
-            "phpstan/extension-installer": true,
-            "cweagans/composer-patches": true,
```

Comment:
running composer install locally got notice:

```
➜  getrector.org git:(tv-laravel-11) composer install
cweagans/composer-patches contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "cweagans/composer-patches" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?] n
```

I guess it can be keep here to avoid it composer install not executed on server while files already added.

Reply:
Remove vendor and run composer update seems solve it. I will verify once more once this PR deployed.